catboost/libs/metrics/ut/fairloss_ut.cpp

```diff
@@ -0,0 +1,52 @@
+#include <library/unittest/registar.h>
```

Comment:
Please add to your pr two python files with a stress test for every metric.
Stress test must compare results of your implementation with some other existing implementation, for example, one from sklearn.metrics or some other one.

To use your metric from python you can use eval_metric function.

Please provide the output of your stress test in the comment.

Reply:
Should I add this stress tests to somewhere in the repository or is it enough to run it with my local build and just add them here as a file, providing result in comment?